linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_clients.js

```diff
@@ -0,0 +1,105 @@
+var RouterClient = (function() {
+  var template;
+
+  function getMetricDefinition(routerName, clientName) {
+    return [
+      {
+        description: "Requests",
+        metricSuffix: "requests",
+        query: Query.clientQuery().withRouter(routerName).withClient(clientName).withMetric("requests").build()
```

Comment:
True, I've seen the jQuery style too in other contexts - it was just the `with` that tripped me up here. Maybe I'm used to the usage in Finagle?